charts/activemq/Chart.yaml

```diff
@@ -7,7 +7,7 @@ keywords:
 name: activemq
 sources:
   - https://github.com/Alfresco/alfresco-helm-charts
-version: 3.4.1
+version: 3.4.2
```

Comment:
we probably want to bump the minor version (the second digit) given that we are supporting a new thing that was not supported before, and not really a bug fix (something that was supposed to be working but it was not)